src/SFA.DAS.ApplyService.Database/Tables/Apply.sql

```diff
@@ -24,6 +24,7 @@
     [Assessor1ReviewStatus] NVARCHAR(20) NULL, 
     [Assessor2ReviewStatus] NVARCHAR(20) NULL,
     [NotRequiredOverrides] NVARCHAR(MAX) NULL, 
+    [ModerationStatus] NVARCHAR(20) NOT NULL DEFAULT 'New', 
     [OversightStatus] NVARCHAR(20) NOT NULL DEFAULT 'New', 
     [ApplicationDeterminedDate] DATETIME2 NULL
```

Comment:
probably a good idea to get this addition in a little branch of its own and get it into master ASAP so it doesn't break test releases of other branches

Reply:
this has been done